.github/workflows/publish-npm-packages.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish npm packages
+
+on:
+    workflow_dispatch:
+        inputs:
+            wp_version:
+                description: 'WordPress major version'
+                required: true
+                type: string
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    wp-release:
+        name: WordPress major bugfix release
+        runs-on: ubuntu-latest
+        environment: WordPress packages
+        steps:
+            - name: Checkout (for CLI)
+              uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+              with:
+                  path: main
+                  ref: trunk
```

Comment:
Shouldn't this point to the release branch, e.g. `wp/6.0`?

Reply:
The script that runs takes care of selecting a proper branch here:

https://github.com/WordPress/gutenberg/blob/d52b240cbb0e5c5e1bf6789629b30121daac1615/bin/plugin/commands/packages.js#L63-L75

It uses `--depth=100` to ensure that the previous publish commit can be read by Lerna.

Reply:
Cool! Let's add that as a comment in the workflow for the next person having the same question :-)

Reply:
Good point, I will add it in both jobs 👍🏻